### DISCUSSION TITLE
Additional tag <script> and <link> only in some fragments/pages

### DISCUSSION THREAD

**Discussion by manlioesp** [unpoly/unpoly#271] [Q&A]
Hi! I'm discovering unpoly and I'm really happy for the possibilities it gives for non Javascript coders like me.
Usually when I build pages in PHP or HTML I append some additional files only on specific pages in the <head> tag like this:

![code](https://user-images.githubusercontent.com/9567588/131345373-e82fcf0f-8579-4d06-9f98-6e8ce16bd5d7.jpg)

I have read the documentation and I found this https://unpoly.com/legacy-scripts about inline <script>, but I don't know how to apply to my case both for css and js scripts. 
Could someone provide an example or can give me some advice for the best practice in unpoly to solve my problem?
Thank you very much.

**Comment by adam12** [comment]
Did you notice an issue with inline `<link>` tags? If you're willing to accept the risks of inline `<scrip>` tags, I'd just follow the instructions in the link you provided to enable that feature. I'm fairly sure inline `<link>` tags just work, without any unnecessary changes. Please correct me if I'm wrong.

There's also this guide: https://makandracards.com/makandra/52361-unpoly-loading-large-libraries-on-demand

I'd personally just load everything, all the time, unless it was absolutely enormous. With proper caching, it's likely simpler.

**Reply by manlioesp** [reply]
Thank you Adam for your quick reply. 
When I work in PHP, I usually put <link> tag in the <head> tag but when I load partial I can't do this.  So I was asking only for this reason.
Your link seems very useful.

> I'd personally just load everything, all the time, unless it was absolutely enormous. With proper caching, it's likely simpler.

I will follow this advice and forget excessive optimization :-).
Thank you very much and have a good day.
